app/views/admin/pages/dashboard.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1>Admins Dashboard</h1>
+
+<div class="card text-white bg-dark mb-3" style="max-width: 18rem;">
+  <div class="card-header">Admins Panel</div>
+  <div class="card-body">
+    <h5 class="card-title">Creating Kudo</h5>
+    <p class="card-text">Some quick example text to build on the card title and make up the bulk of the card's content.</p>
+  </div>
+</div>
```

Comment:
why we have two the same vies for dasboard?

Reply:
Right, totally unnecessary view on the way back when renaming from admin to admins